Extensions/xkit_preferences.js

```diff
@@ -105,23 +105,32 @@ XKit.extensions.xkit_preferences = new Object({
 
 		// Check and deliver initial messages.
 		if (XKit.storage.get("xkit_preferences","initial_mail_sent","0") === "0") {
-			var randomnumber = 1000 + Math.floor(Math.random()*100000);
-			XKit.extensions.xkit_preferences.news.create(91111, "Welcome to XKit!",
-				"<h1>Welcome, and thanks for installing XKit 7!</h1> In this panel, you will receive news and updates on XKit 7. "+
-				"These include, but not limited to, new features, bug fixes and things you should do if you experience problems with your XKit."+
-				"<h2>Learn XKit</h2> Clicking on the My XKit tab will give you a list of all the extensions you have installed. "+
-				"You can read their descriptions, and click on <strong>more information</strong> link below their description (if available) "+
-				"to learn even more about them and how to use them."+
-				"<h2>Customize XKit</h2> Nearly all extensions of XKit has settings that you can customize: from appearance to custom tags, "+
-				"you can toggle and change their settings from the My XKit tab."+
-				"<h2>Expand XKit</h2> XKit automatically installs some default extensions, but if you want more, you can check the extension "+
-				"gallery for more. To do that, just click on the <strong>Get Extensions</strong> tab on the bottom of this window."+
-				"<h2>Help XKit</h2> XKit is free of charge, and I'm not making any money off it in any way. "+
-				"If you are using XKit for the first time, give it a try for a few days, and if you like it, "+
-				"please donate by going to the About tab on this window to support free software. "+
-				"You can also help by sharing XKit with your followers and friends, and spreading the word"+
-				"<h2>Thanks for reading!</h2> Again, thanks for installing XKit, and I hope you enjoy using it!<br><br>"+
-				"<em>Yours faithfully,<br>Xenixlet #" + randomnumber + "<br>Your Personal Xenixlet, XKit Assistant.</em>");
+			var initial_mail = {
```

Comment:
What is the benefit to embedding this mail instead of distributing it through the actual Paperboy system?  If they have connectivity enough to install XKit, surely they can pull Paperboy messages.